content/accessible/labels-and-text-alternatives/index.md

```diff
@@ -0,0 +1,334 @@
+---
+page_type: guide
+title: Labels and text alternatives
+author: robdodson
+description: |
+  In order for a screen reader to present a spoken UI to the user, meaningful
+  elements must have proper labels or text alternatives. A label or text
+  alternative gives an element its accessible name, one of the key properties
+  for expressing element semantics in the accessibility tree.
+web_lighthouse: 
+  - label
+  - image-alt
+  - input-image-alt
+  - frame-title
+  - object-alt
+  - document-title
+  - button-name
+  - link-name
+wf_blink_components: N/A
+---
+
+# Labels and text alternatives
+
+In order for a screen reader to present a spoken UI to the user, meaningful
+elements must have proper labels or text alternatives. A label or text
+alternative gives an element its accessible **name**, one of the key properties
+for [expressing element semantics in the accessibility
+tree](https://web.dev/accessible/semantics-and-screen-readers/#semantic-properties-and-the-accessibility-tree).
+
+When an element's name is combined with the element's **role**, it gives the
+user context so they can understand what type of element they're interacting
+with, and how it is represented on the page. If a name is not present, then a
+screen reader will only announce the element's role. Imagine trying to navigate
```

Comment:
Small nit-- across the guide, you use future tense occasionally ('will'). Stylistic, we are trying to stick to active.

So here-- instead of 'screen reader will only announce', you can say 'screen reader only announces'. It's similar, and a bit more direct.

Reply:
Good to sweep guide for 'will'.